uint64 timestamp
float32[3] kR
float32[3] kOm
